topic_folders/maintainers/maintainers.md

```diff
@@ -91,4 +91,44 @@ the onboarding process.
 [SWC GH Lesson Maintainer Teams]: https://github.com/orgs/swcarpentry/teams/lesson-maintainers
 [DC GH Lesson Maintainer Teams]: https://github.com/orgs/datacarpentry/teams/lesson-maintainers
 [LC GH Lesson Maintainer Teams]: https://github.com/orgs/librarycarpentry/teams/lesson-maintainers
-[The Carpentries GH Lesson Maintainer Teams]: https://github.com/orgs/carpentries/teams/lesson-maintainers
\ No newline at end of file
+[The Carpentries GH Lesson Maintainer Teams]: https://github.com/orgs/carpentries/teams/lesson-maintainers
+
+
+## Maintainer Community Meetings
+
+The Maintainer community meets on the third Wednesday of every month. Times for these meetings can be found on the [Community Calendar](https://static.carpentries.org/community/#community-events). A reminder for these meetings should be sent out to the Maintainers email list one week before the meeting takes place, and the [Maintainers Etherpad](https://pad.carpentries.org/maintainers) should also be updated. Ideally, these meetings will be co-facilitated by two members of the Carpentries team.
```

Comment:
Could we change "two members of the Carpentries team" to "a member of [The Carpentries Core Team](https://carpentries.org/team/) and the Maintainer Community Development Lead"?